Assets/HoloToolkit/UI/Shaders/3DTextShader.shader

```diff
@@ -70,20 +72,22 @@
 
                 v2f vert (appdata_t v)
                 {
+                    UNITY_SETUP_INSTANCE_ID(v);
                     v2f o;
                     o.vertex = mul(UNITY_MATRIX_MVP, v.vertex);
                     o.texcoord = TRANSFORM_TEX(v.texcoord, _MainTex);
                     o.color = v.color;
                     #ifdef UNITY_HALF_TEXEL_OFFSET
                     o.vertex.xy += (_ScreenParams.zw-1.0)*float2(-1,1);
                     #endif
+                    UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(o);
                     return o;
                 }
 
                 half4 frag (v2f i) : COLOR
                 {
                     half4 col = i.color;
-                    col.a *= tex2D(_MainTex, i.texcoord).a;
+                    col.a *= tex2D(_MainTex, UnityStereoScreenSpaceUVAdjust(i.texcoord, _MainTex_ST)).a;
```

Comment:
As far as testing, you *should* be able to just set the stereo method to "Single Pass Instanced" and through a 3DTextPrefab into the scene. That being said, I wasn't able to reproduce the problem in my current environment (5.6b10). It was reported earlier this morning in Slack and I was filing the bug on their behalf.  

I can throw 5.5 back on and try to reproduce there, but I might not be able to get around to that tonight.